scripts/primary-map-features.js

```diff
@@ -13,20 +13,42 @@ if (!argv.output) {
   console.log('  OPTIONS');
   console.log('    --output counts.csv');
   console.log('');
-  return;
+  throw new Error('USAGE: error');
```

Comment:
Is it a general best practice to throw an error? What I am thinking is the old version was cleaner.

### Earlier
```
$ node scripts/primary-map-features.js

USAGE: node primary-map-features.js OPTIONS

  OPTIONS
    --output counts.csv
```

### Now
```
$ node scripts/primary-map-features.js

USAGE: node primary-map-features.js OPTIONS

  OPTIONS
    --output counts.csv


/Users/demo/repos/mapbox/osm-compare/scripts/primary-map-features.js:16
  throw new Error('USAGE: error');
        ^
Error: USAGE: error
    at Object.<anonymous> (/Users/demo/repos/mapbox/osm-compare/scripts/primary-map-features.js:16:9)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:945:3
```